src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+                enterprise: 'Annual',
+            },
+            {
+                label: 'paymentMethod',
+                business: 'Invoice',
+                enterprise: 'Invoice',
+            },
+        ],
+    },
+]
+
+/** Feature info dictionaries */
+
+const CIP_FEATURE_INFO: Record<string, FeatureInfo> = {
+    codeSearch: {
+        label: 'Code Search',
+        description: 'Super-fast, intuitive, and powerful code search across your entire codebase.',
+    },
+    codeNavigation: {
+        label: 'Code navigation',
+        description:
+            "Traverse your entire codebase with precise code navigation for cross-repository 'Go to definition,' 'Find references,' and more.",
+    },
+    batchChanges: {
+        label: 'Batch Changes',
+        description: 'Apply and track large-scale code changes across all of your repositories and code hosts.',
+    },
+    codeInsights: {
+        label: 'Code Insights',
+        description:
+            'Track and visualize trends in your entire codebase with visualizations that are kept automatically up to date.',
+    },
+    notebooks: {
+        label: 'Notebooks',
+        description: 'Create living documentation that interacts directly with your code.',
+    },
+    codeMonitoring: {
+        label: 'Code monitoring',
+        description: 'Get alerts when changes are made to your codebase.',
+    },
+    comprehensiveApi: {
+        label: 'Comprehensive API',
+        description: 'A secure, robust GraphQL API for your repository and code data.',
+    },
+}
+
+const SECURITY_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ssoSaml: {
+        label: 'SSO/SAML',
+        description: 'Single sign-on user authentication with SAML, OAuth, OpenID Connect, and HTTP auth proxy.',
+    },
+    securityRoles: {
+        label: 'User and admin roles',
+        description:
+            'Allow only certain users (site admins) to view and edit site configuration and repository/code host credentials.',
+    },
+    repoPerms: {
+        label: 'Standard repository permissions',
+        description:
+            'Apply the repository permissions from your code host to restrict which repositories a user can search and browse.',
+    },
+    customPerms: {
+        label: 'Custom repository permissions',
+        description:
+            'Apply the repository permissions from your code host or use the explicit permissions API to restrict which repositories a user can search and browse.',
+    },
+    privateInstance: {
+        label: 'Private instance access',
+        description: 'Your instance is only accessible from allowlisted IPs. ',
+    },
+    analytics: {
+        label: 'In-product analytics',
+        description: 'Understand user engagement and calculate the value of using Sourcegraph.',
+    },
+}
+
+const DEPLOYMENT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    cloudDeployment: {
+        label: 'Secure and single-tenant Cloud deployment',
```

Comment:
Same point as above for removing the "and" and maybe removing "secure"

Reply:
@st0nebraker - "Single-tenant Cloud deployment"